### PR TITLE
General default configuration fixes

### PR DESCRIPTION
**Pull request by reset**
This PR contains some fixes for the default configuration of a builder-api service. We only need to specify the configurables from the web portion and any *required* values inside a `user.toml` or `default.toml`. Proper default values are provided by the Rust code itself otherwise!

* builder-api once again only specifies builder-web configuration in default.toml
* move make support files for web-dev configuration into support directory
* use Github client_id/secret from Habitat organization instead of my personal org

### CODE REVIEW

**Comment by thesentinels** [comment]
By analyzing the blame information on this pull request, we identified @metadave, @smith, @fnichol, @adamhjk and @jtimberman to be potential reviewers

File: Makefile [hunk 1]
```diff
@@ -109,8 +109,8 @@ start-web-dev:
 	@echo "==> Starting builder-api on 9636"
 	$(web_dev_run) sh -c 'mkdir -p /hab/svc/hab-director; \
  	mkdir -p /hab/svc/hab-builder-api; \
-	cp /src/components/builder-web/builder_web.toml /hab/svc/hab-director/user.toml; \
-	cp /src/components/builder-api/builder_api.toml /hab/svc/hab-builder-api/user.toml; \
+	cp /src/support/builder-director-dev.toml /hab/svc/hab-director/user.toml; \
+	cp /src/support/builder-api-dev.toml /hab/svc/hab-builder-api/user.toml; \
```
Comment (metadave) [inline Makefile]: I like this, it's cleaner

File: components/builder-api/src/config.rs [hunk 2]
```diff
@@ -28,10 +28,10 @@ const GITHUB_URL: &'static str = "https://api.github.com";
 // Default Client ID for providing a default value in development environments only. This is
 // associated to Jamie Winsor's GitHub account and is configured to re-direct and point to a local
 // builder-api.
-const DEV_GITHUB_CLIENT_ID: &'static str = "e98d2a94787be9af9c00";
+const DEV_GITHUB_CLIENT_ID: &'static str = "0c2f738a7d0bd300de10";
 // Default Client Secret for development purposes only. See the `DEV_GITHUB_CLIENT_ID` for
 // additional comments.
-const DEV_GITHUB_CLIENT_SECRET: &'static str = "e5ff94188e3cf01d42f3e2bcbbe4faabe11c71ba";
+const DEV_GITHUB_CLIENT_SECRET: &'static str = "438223113eeb6e7edf2d2f91a232b72de72b9bdf";
```
Comment (metadave) [inline components/builder-api/src/config.rs]: how often do these expire?

**Comment by metadave** [comment]
nice!

gif-keyboard-6427982498781438707

File: components/builder-api/src/config.rs (hunk 2 above)
Comment (reset) [inline components/builder-api/src/config.rs]: Never! They're from an OAuth application on the Habitat organization called "builder-dev". It's just existing for us to be able to redirect to our dev machines upon login

**Comment by metadave** [comment]
@thesentinels r+

**Comment by thesentinels** [comment]
:pushpin: Commit a61b1a5 has been approved by `metadave`

**Comment by thesentinels** [comment]
:hourglass: Testing commit a61b1a5 with merge 5ea06d9...

**Comment by thesentinels** [comment]
:sunny: Test successful - [travis](https://travis-ci.org/habitat-sh/habitat/builds/139308760)

**Pull request merged**